src/main/resources/static/css/application.css

```diff
@@ -45,8 +41,37 @@
 .content-container {
     display: flex;
     flex-direction: column;
-
+    height:100%;
     margin: 2rem 4rem;
-
     font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
+
+.register-form {
+    display: flex;
+    justify-content:center;
+    font-size: 30px;
+}
+
+.register-heading {
+    text-align: center;
+    margin-top: 4rem;
+    font-size: 40px;
+}
+
+.register-inputs {
+    width:75%;
+    display: flex;
+    flex-direction:column;
+    margin-bottom:1rem;
+}
+
+.input-container {
+    width:50%;
+    display:flex;
+    flex-direction:column;
+    align-items: center;
+}
+
+.register-textfields {
+    height:2rem;
+}
```

Comment:
Please include one newline at the bottom (note the angry red ⛔ symbol) :') This is because text files on *nix platforms are strictly only text files if and only if they end with a `\n` at the end